servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractResteasyV1Test.java

```diff
@@ -121,6 +122,17 @@ public void testBasic() {
             .as(Branch.class);
     Assertions.assertNotEquals(newReference.getHash(), commitResponse.getHash());
 
+    EntriesResponse entries =
+        rest()
+            .get("trees/tree/{branch}/entries", newReference.getName())
+            .then()
+            .statusCode(200)
+            .extract()
+            .as(EntriesResponse.class);
+    assertThat(entries.getEntries())
+        .hasSize(1)
+        .allSatisfy(e -> assertThat(e.getContentId()).isNull());
```

Comment:
Do you think a plain JSON string test like `testCommitMetaAttributes` for the new content ID attribute is worth adding (in addition to this test, or replacing this test)? My thinking is that java code blurs the case of having a `null` attribue and not having an attribute.